integration_tests/pages/page.ts

```diff
@@ -9,26 +10,32 @@ export default abstract class Page {
 
   constructor(
     private readonly title: string,
-    private readonly options: { axeTest?: boolean } = {
+    private readonly options: { axeTest?: boolean; axeRulesToIgnore?: string[] } = {
```

Comment:
Copied over from Staff UI - ability to ignore accessibility rules for known issues